src/gui/qgisinterface.h

```diff
@@ -534,6 +534,24 @@ class GUI_EXPORT QgisInterface : public QObject
     virtual QAction *actionCheckQgisVersion() = 0;
     virtual QAction *actionAbout() = 0;
 
+    // Shape digitize actions
+    virtual QAction *actionCircle2Points()  = 0;
```

Comment:
Can you add fix for all these?

Reply:
Done Thanks.

> "set the default pan map tool." ? :wink:

I don't know what you're talking about. :rofl: